apps/create-fuels-counter-guide/src/pages/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { Button } from "@/components/Button";
+import toast from "react-hot-toast";
+import { useActiveWallet } from "@/hooks/useActiveWallet";
+import useAsync from "react-use/lib/useAsync";
+
+const contractId = contractIds.testContract;
+
+const hasContract = process.env.NEXT_PUBLIC_HAS_CONTRACT === "true";
+const hasPredicate = process.env.NEXT_PUBLIC_HAS_PREDICATE === "true";
+const hasScript = process.env.NEXT_PUBLIC_HAS_SCRIPT === "true";
+
+export default function Home() {
+  const { wallet, walletBalance, refreshWalletBalance } = useActiveWallet();
+  const [contract, setContract] = useState<TestContractAbi>();
+  const [counter, setCounter] = useState<number>();
+
+  /**
+   * useAsync is a wrapper around useEffect that allows us to run asynchronous code
+   * See: https://github.com/streamich/react-use/blob/master/docs/useAsync.md
+   */
+  useAsync(async () => {
+    if (hasContract && wallet) {
+      const testContract = TestContractAbi__factory.connect(contractId, wallet);
+      setContract(testContract);
+      const { value } = await testContract.functions.get_count().get();
+      setCounter(value.toNumber());
+    }
+  }, [wallet]);
+
+  // eslint-disable-next-line consistent-return
+  const onIncrementPressed = async () => {
+    if (!contract) {
+      return toast.error("Contract not loaded");
+    }
+
+    if (walletBalance?.eq(0)) {
+      return toast.error(
+        "Your wallet does not have enough funds. Please click the 'Top-up Wallet' button in the top right corner, or use the local faucet.",
+      );
+    }
+
+    const { value } = await contract.functions.increment_counter(bn(1)).call();
+    setCounter(value.toNumber());
+
+    await refreshWalletBalance?.();
+  };
+
+  // #region create-fuels-counter-guide-on-decrement-react-function
+  const onDecrementPressed = async () => {
+    if (!contract) {
+      return toast.error("Contract not loaded");
+    }
+
+    if (walletBalance?.eq(0)) {
+      return toast.error(
+        "Your wallet does not have enough funds. Please click the 'Top-up Wallet' button in the top right corner, or use the local faucet.",
+      );
+    }
+
+    const { value } = await contract.functions.decrement_counter(bn(1)).call();
+    setCounter(value.toNumber());
+
+    await refreshWalletBalance?.();
+  };
+  // #endregion create-fuels-counter-guide-on-decrement-react-function
+
+  return (
+    <>
+      <div className="flex gap-4 items-center">
+        <FuelLogo />
+        <h1 className="text-2xl font-semibold ali">Welcome to Fuel</h1>
+      </div>
+
+      {hasContract && (
+        <span className="text-gray-400">
+          Get started by editing <i>sway-programs/contract/main.sw</i> or{" "}
+          <i>src/pages/index.tsx</i>.
+        </span>
+      )}
+
+      <span className="text-gray-400">
+        This template uses the new{" "}
+        <Link href="https://fuellabs.github.io/fuels-ts/tooling/cli/fuels/">
+          Fuels CLI
+        </Link>{" "}
```

Comment:
```suggestion
        </Link>
```

Now i'm thinking there is a reason behind this?

Reply:
Probably the formatter or eslint